sailpoint.sgnl.yaml

```diff
@@ -0,0 +1,1575 @@
+    pageSize: 100
+    pagesOrderedById: false
+    attributes:
+      - name: instance
+        externalId: instance
+        type: String
+      - name: name
+        externalId: name
+        type: String
+      - name: ruleName
+        externalId: ruleName
+        type: String
+      - name: key
+        externalId: key
+        type: String
+
+  # TODO: Identify unique and index attributes
+  ObjectAttributeTargets:
+    parent: objectAttributes
+    displayName: SailpointObjectAttributeTargets
+    externalId: attributeTargets
+    description: Entity representing the attribute targets associated with a Sailpoint Identity IQ ObjectConfig.
+    syncFrequency: HOURLY
+    syncMinInterval: 1
+    apiCallFrequency: SECONDLY
+    apiCallMinInterval: 1
+    pageSize: 100
+    pagesOrderedById: false
+    attributes:
+      - name: instance
+        externalId: instance
+        type: String
+      - name: name
+        externalId: name
+        type: String
+      - name: ruleName
+        externalId: ruleName
+        type: String
+      - name: key
+        externalId: key
+        type: String
+      - name: provisionAllAccounts
+        externalId: provisionAllAccounts
+        type: Bool
+
+  PolicyViolation:
+    displayName: SailpointPolicyViolation
+    externalId: PolicyViolations
+    description: Entity representing a Sailpoint Identity IQ Policy Violation.
+    syncFrequency: HOURLY
+    syncMinInterval: 1
+    apiCallFrequency: SECONDLY
+    apiCallMinInterval: 1
+    pageSize: 100
+    pagesOrderedById: false
+    attributes:
+      - name: id
+        externalId: id
+        type: String
+        indexed: true
+        uniqueId: true
+      - name: policyName
+        externalId: policyName
+        type: String
+      - name: constraintName
+        externalId: constraintName
+        type: String
+      - name: identityValue
+        externalId: identity.value
+        type: String
+      - name: identityRef
+        externalId: identity..["$ref"]
+        type: String
+      - name: identityDisplayName
+        externalId: identity.displayName
+        type: String
+      - name: ownerValue
+        externalId: owner.value
+        type: String
+      - name: ownerRef
+        externalId: owner.["$ref"]
+        type: String
+      - name: ownerDisplayName
+        externalId: owner.displayName
+        type: String
+      - name: description
+        externalId: description
+        type: String
+      - name: status
+        externalId: status
+        type: String
+
+      # Meta
+      - name: resourceType
+        externalId: $.meta.resourceType
+        type: String
+      - name: created
+        externalId: $.meta.created
+        type: DateTime
+      - name: lastModified
+        externalId: $.meta.lastModified
+        type: DateTime
+      - name: version
+        externalId: $.meta.version
+        type: String
+      - name: location
+        externalId: $.meta.location
+        type: String
+
+  Roles:
```

Comment:
Can we be consistent with keeping internal names singular / plural? It seems to be all over the place at the moment